Makefile

```diff
@@ -41,14 +42,19 @@ certificates/domain.key:
 
 .PHONY: up-local
 up-local: .install-fqdn certificates/domain.crt certificates/domain.key .create-secrets ## deploy osparc ops stacks and simcore
-	bash scripts/local-deploy.sh
+	bash scripts/deployements/local-deploy.sh
```

Comment:
``deployEments`` or ``deployments`` (w/o E)?

Reply:
That was a nice frenglish word...